debugInterface.js

```diff
@@ -23,12 +21,25 @@ module.exports = class DebugInterface {
 
   get exportTable () {
     return {
-      'print': function (offset, length) {
+      'print': function (offset = 0, length = 100) {
```

Comment:
I think these shouldn't have defaults (the wasm interface (ECI) mandates them as two parameters).